numcodecs/zarr3.py

```diff
@@ -25,6 +25,19 @@
 
 from __future__ import annotations
 
+# Short workaround for skipping the doctest above in a WASM environment
+# compiled via Emscripten where threads are not available, and accessing
+# the pytest config has no effect and ignoring warnings does not work.
+try:
+    import pytest
```

Comment:
What would it take to not import pytest here?

Reply:
Last time I checked, we would need a way to run this doctest and not raise a warning that gets converted to an error. The other way could be to not add this doctest here, which is not the best way to proceed.